ui/src/pages/frontsite/components/Series/SeriesLiterature.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Divider } from 'semantic-ui-react';
+
+export class SeriesLiterature extends Component {
+  render() {
+    return <Divider horizontal>Literature in this series</Divider>;
```

Comment:
please leave a comment next time that you will continue on this section, I was confused for a moment why there is only a divider :)

Reply:
Ahh, sure. I left a comment in the PR message but it's very easy to miss so it's better to have it in the code.